### PR TITLE
Add revert_in parameter to napalm_configure (replaces #32)

### PR DESCRIPTION
**Pull request by gneville-ot**
Extends what was proposed in #32 but adds the ability to confirm a commit that was sent previously with a revert_in timer.

### CODE REVIEW

**Comment by ktbyers** [comment]
I will also see if I can fix the CI-CD issues...they look unrelated to your PR.

**Comment by dbarrosop** [comment]
Yes, the CI issues are completely unrelated. If you don't beat me to it I may try to do it myself by the end of next week. My schedule is crazy nowadays :(

**Comment by ubaumann** [comment]
The CI issue should be easy to solve.

https://github.com/nornir-automation/nornir_napalm/blob/master/.github/workflows/main.yaml#L55 
I believe you need to use `snok/install-poetry@v1` instead of `snok/install-poetry@v1.1.1`. It works with `v1` in the linter job, and I have other pipelines using `v1`.

If you both are busy I can make a PR

**Comment by ktbyers** [comment]
@ubaumann I will see if I can get to it this weekend...probably won't get to it before then.

File: nornir_napalm/plugins/tasks/napalm_configure.py [hunk 1]
```diff
@@ -38,9 +44,9 @@ def napalm_configure(
     dry_run = task.is_dry_run(dry_run)
     if not dry_run and diff:
         if commit_message:
-            device.commit_config(message=commit_message)
+            device.commit_config(message=commit_message, revert_in=revert_in)
         else:
-            device.commit_config()
+            device.commit_config(revert_in=revert_in)
```
Comment (ktbyers) [inline nornir_napalm/plugins/tasks/napalm_configure.py:49]: We should just add some logic here to only use the `revert_in` argument if it not None. This is probably less risky for any case where someone is using a NAPALM driver/community driver in some way that has not implemented `revert_in`.

**Comment by ktbyers** [comment]
Looks good with the caveat of the one comment I made regarding the need of logic of the form `if revert_in is not None`.

Also needs to be implemented in conjunction with adding `rollback` support to nornir-napalm

**Comment by ktbyers** [comment]
Superseded by https://github.com/nornir-automation/nornir_napalm/pull/44 to fix CI-CD issues.

**Pull request closed without merging**